datastore/in_memory/member.go

```diff
@@ -3,33 +3,67 @@ package in_memory
 import (
 	"errors"
 	"memberserver/api/models"
+	"sort"
 )
 
 var Members = map[string]models.Member{}
 
 func (i In_memory) GetTiers() []models.Tier {
-	return []models.Tier{}
+	return i.Tiers
 }
 func (i In_memory) GetMembers() []models.Member {
-	return []models.Member{}
+	return MemberMapToSlice(i.Members)
 }
 func (i In_memory) GetMemberByEmail(email string) (models.Member, error) {
-	for _, k := range testUsers {
+	for _, k := range i.Members {
 		if k.Email == email {
 			return k, nil
 		}
 	}
 	return models.Member{}, errors.New("error getting user: not found")
 }
 func (i In_memory) AssignRFID(email string, rfid string) (models.Member, error) {
-	return models.Member{}, nil
+	if len(rfid) == 0 {
+		return models.Member{}, errors.New("not a valid rfid")
+	}
+
+	for _, member := range i.Members {
+		if member.Email != email {
+			continue
+		}
+		return member, nil
+	}
+	return models.Member{}, errors.New("user not found")
 }
 func (i In_memory) AddNewMember(newMember models.Member) (models.Member, error) {
-	return models.Member{}, nil
+	return newMember, nil
 }
 func (i In_memory) AddMembers(members []models.Member) error {
 	return nil
 }
 func (i In_memory) GetMembersWithCredit() []models.Member {
 	return []models.Member{}
 }
+func (i In_memory) ProcessMember(newMember models.Member) error {
+	return nil
+}
+
+func MemberMapToSlice(m map[string]models.Member) []models.Member {
+	var members []models.Member
+
+	for _, member := range m {
+		members = append(members, member)
+	}
+
+	sort.Sort(ByID(members))
+
+	println(members)
```

Comment:
Why are you doing `println` here? To see if it sorts?  If so, remove it. Also, why are you sorting by Id for member?